src/task-runtime.ts

```diff
@@ -35,7 +35,7 @@ export class TaskRuntime {
   public readonly workdir: string;
 
   constructor(workdir: string) {
-    this.workdir = resolve(workdir);
+    this.workdir = workdir;
```

Comment:
Let's keep other changes to a minimum. Who knows where else we use `TaskRuntime` with a relative path.
```suggestion
    this.workdir = resolve(workdir);
```

Reply:
I agree 👍